build-index-page.sh

```diff
@@ -3,13 +3,27 @@
 set -euo pipefail
 
 bucket="${1?Missing bucket name argument}"
+folder="${2-simple}"
+
+# Strip any trailing slashes
+bucket="${bucket%/}"
+folder="${folder%/}"
 
 "$(dirname "$0")/gsutil-auth-helper.sh"
 
-(
-  echo '<!DOCTYPE html><html><head><title>Astronomer Python Packages</title></head><body><pre>'
-  gsutil ls -lh "gs://$bucket/simple/" | sed '/\/$\|index.html$/d; s!gs://[^/]\+/simple/\(.*\)$!<a href="\1">\1</a>!'
-  echo '</pre></body></html>'
-) > index.html
+while [[ -n "$folder" ]]; do
+  (
+    echo '<!DOCTYPE html><html><head><title>Astronomer Python Packages</title></head><body><pre>'
```

Comment:
does it make sense to have html file? instead of hardcode string?
in case if somebody from UI team would love to add styles or so?

Reply:
Could do, but then we have to worry about how we insert the template values.

For reference here is what the pip PEP503 repo looks like https://pypi.org/simple/apache-airflow/ :)

I'd say lets do that when we have a specific need.